scripts/test_execution.json

```diff
@@ -27,28 +27,28 @@
                 "Name": "vm2"
```

Comment:
Should this line and lines above also change?

Reply:
Hmm, I can change them now as well. These will become temporarily inactive post-merge anyway, so I can re-label. Can use that time to recreate the runners with new names too.